src/App.css

```diff
@@ -7,3 +7,16 @@
 .cdc-react .dropdown-items {
   overflow: scroll;
 }
+
+.cdc-react .sort-arrows {
+  margin-left: 5px; /* Hack until we replace the sort icons */
+  margin-right: 5px; /* Hack until we replace the sort icons */
```

Comment:
Can you add a TODO or FIX to the comment so it gets picked up by the IDE to address later?